internal/service/streamconnection/resource_stream_connection.go

```diff
@@ -67,6 +70,16 @@ var ConnectionSecurityObjectType = types.ObjectType{AttrTypes: map[string]attr.T
 	"protocol":                  types.StringType,
 }}
 
+type TFDbRoleToExecuteModel struct {
```

Comment:
same, why TF?

Reply:
following convention from other methods. Also, it signifies we are working with Terraform model specifically. I believe we often follow this convention in our resources